presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -57,6 +57,7 @@
     private final long outputPositions;
 
     private final DataSize physicalWrittenDataSize;
+    private final DataSize spilledDataSize;
```

Comment:
Move this down to be after the `peak` stats (here and everywhere else in this class)